docs/source/whatsnew.rst

```diff
@@ -1,126 +1,190 @@
-What's new in the v2.0 Alpha
-============================
-
-A word about the Alpha release
-------------------------------
-
-The Alpha release of Hyperledger Fabric v2.0 allows users to try out two exciting
-new features --- the new Fabric chaincode lifecycle and FabToken. The Alpha release
-is being offered to provide users a preview of new capabilities and is not meant
-to be used in production. Additionally there is no upgrade support to the v2.0
-Alpha release, and no intended upgrade support from the the Alpha release
-to future versions of v2.x.
-
-Fabric chaincode lifecycle
---------------------------
-
-The Fabric 2.0 Alpha introduces decentralized governance for chaincode, with
-a new process for installing a chaincode on your peers and starting it
-on a channel. The new Fabric chaincode lifecycle allows
-multiple organizations to come to agreement on the parameters of a chaincode,
-such as the chaincode endorsement policy, before it can be used to interact
-with the ledger. The new model offers several improvements over the previous
-lifecycle:
-
-* **Multiple organizations must agree to the parameters of a chaincode:** In
-  the release 1.x versions of Fabric, one organization had the ability to set
-  parameters of a chaincode (for instance the endorsement policy) for all other
-  channel members. The new Fabric chaincode lifecycle is more flexible since
-  it supports both centralized trust models (such as that of the previous
-  lifecycle model) as well as decentralized models requiring a sufficient number
-  of organizations to agree on an endorsement policy before it goes into effect.
-
-* **Safer chaincode upgrade process:** In the previous chaincode lifecycle,
-  the upgrade transaction could be issued by a single organization, creating a
-  risk for a channel member that had not yet installed the new chaincode. The
-  new model allows for a chaincode to be upgraded only after a sufficient
-  number of organizations have approved the upgrade.
-
-* **Easier endorsement policy updates:** Fabric lifecycle allows you to change
-  an endorsement policy without having to repackage or reinstall the chaincode.
-  Users can also take advantage of a new default policy that requires endorsement
-  from a majority of members on the channel. This policy is updated automatically
-  when organizations are added or removed from the channel.
-
-* **Inspectable chaincode packages:** The Fabric lifecycle packages chaincode in
-  easily readable tar files. This makes it easier to inspect the chaincode
+What's new in the v2.0 Beta
```

Comment:
How about: 

What's new in Fabric v2.0 Beta release

(because it's nice to associate it as 'Fabric v2.0' and to state that it's a 'Beta release'